cluster/addons/metadata-agent/stackdriver/metadata-agent.yaml

```diff
@@ -56,6 +56,13 @@ spec:
       dnsPolicy: ClusterFirst
       restartPolicy: Always
       schedulerName: default-scheduler
+      tolerations:
+      - key: "node.alpha.kubernetes.io/ismaster"
```

Comment:
Below you are adding a toleration for all keys with NoSchedule effect, so this one doesn't seem to be doing anything.